pyartcd/pyartcd/pipelines/build_rhcos.py

```diff
@@ -245,8 +127,8 @@ def wait_for_builds(self):
               help="Ignore in-progress builds instead of just exiting like usual")
 @click.option("--new-build", required=False, default=False, type=bool,
               help="Force a new build even if no changes were detected from the last build")
+@click.option("--job", required=False, type=str, default="build",
```

Comment:
Can this be a list of options? like `['build', 'build-node-image']`? Think we can list the types that we support here.

Reply:
I put it in a list in the Jenkins job parameter https://github.com/openshift-eng/aos-cd-jobs/pull/4383/files